CHANGELOG.md

```diff
@@ -2,13 +2,24 @@
 
 See README.md on how to build the hipCUB documentation using Doxygen.
 
+## (Unreleased) hipCUB-2.13.1 for ROCm 5.6.0
```

Comment:
@Beanavil Can you change the ROCm version to 5.7.0 please?  Thank you.

Reply:
Sure! Should be updated now.

Reply:
Thanks!